### ISSUE TITLE
Flag `ird` not used?

### ISSUE THREAD

**Issue by kyrsjo** [SixTrack/SixTrack#284]
According to the documentation, the flag `ird` in the TRAC block has the following effect:
```
ird (integer) Switch for the type of amplitude variation. In case napx = 1 the amplitude nstart is
used.
• ird = 0 : amplitudes are varied between the amplitudes amp(1) and amp0 with equal
increments:
delta = (amp0 − amp(1))/(napx − 1)
• ird = 1 : amplitude variation to find an estimate for the short term dynamic aperture.
The amplitude is increased or decremented corresponding to stable motion or particle loss
respectively. The change of amplitude is reduced each iteration i ≤ (napx − 1) to:
delta = (amp0 − amp(1))/2^i
```

However when reading the code, the variable is read and then ignored... I think the documentation is wrong here?

**Comment by kyrsjo** [comment]
Also, it refers to `the amplitude nstart`, but `nstart` is actually defined in the POST block as
`nstart, nstop (integers) Start and stop turn number for the analysis of the post–processing (0 0 =
all data used).`

**Comment by kyrsjo** [comment]
Fixed by PR #293
